lib/membrane_live_web/channels/event_channel.ex

```diff
@@ -86,13 +89,14 @@ defmodule MembraneLiveWeb.EventChannel do
              {:ok, is_presenter} <-
```

Comment:
Shouldn't we apply the recommendation from credo?
```suggestion
             {:ok, presenter?} <-
```

Reply:
credo only warns about functions names